feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -142,15 +148,21 @@ internal class HomeViewModel @Inject constructor(
             combine(
                 feedingPointStateFlow,
                 feedingStateFlow,
-                feedingRouteStateFlow
-            ) { feedingPointUpdate, feedingUpdate, feedingRouteUpdate ->
+                feedingRouteStateFlow,
+                stateFlow
+            ) { feedingPointUpdate, feedingUpdate, feedingRouteUpdate, homeStateFlow ->
                 updateState {
                     copy(
                         feedingPointState = feedingPointUpdate,
                         feedState = feedingUpdate,
                         feedingRouteState = feedingRouteUpdate,
                     )
                 }
+                if (feedingPointUpdate.feedingPoints.isNotEmpty() &&
```

Comment:
Can we move this first check into the usecase?

Reply:
Moved to usecase